### PR TITLE
Record of looking at search library options

### PR DESCRIPTION
**Pull request by bz2**
This is a bit haphazard but I've had fun playing with the options.

### CODE REVIEW

**Comment by justin2004** [comment]
@bz2 , there is also:
https://github.com/meilisearch/meilisearch

File: records/improving_search.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+* Language: Javascript
+* Last release: [three months ago 6.3.0](https://www.npmjs.com/package/minisearch)
+* Maintainer's photo: Hanging in the park but only into the portrait
+
+Skips bundling a few options like fancy tokenising and processing but provides an interface for you to bring your own. Eyeball code quality looks pretty good, sane looking typescript and lots of inline documentation.
+
+
+## Recommendation
+
+We should try using both flexsearch with various options, and minisearch with some specific transforms we implement.
```
Comment (alightwing) [inline records/improving_search.md:117]: But why?

**Comment by bz2** [comment]
@justin2004 Amazing, thank you, hadn't played with that one yet, I'll add it.

File: records/improving_search.md (hunk 1 above)
Comment (bz2) [reply records/improving_search.md:154]: Have added a bit above that basically highlights it was easy to find how to do some stuff I want in the docs of the two recommendations here.

**Pull request merged**